Cargo.toml

```diff
@@ -72,6 +75,10 @@ async-lock = "3.4.0"
 libc = "0.2.172"
 zeroize = "1.5.7"
 
+# Fuzz deps
+libfuzzer-sys = { version = "0.4.9" }
+arbitrary = { version = "1.4.1", features = ["derive"] }
```

Comment:
nit: for consistency we should:
1) just do `crate = version`
2) only specify features in the crate that is importing the shared dep